.github/workflows/ci.yml

```diff
@@ -18,12 +18,14 @@ jobs:
         id: prepare
         run: |
             TAG=${GITHUB_REF##*/}
+            echo "tag_name=${TAG}" >> >> $GITHUB_OUTPUT
```

Comment:
You sure about the double ">> >>"?

Reply:
> You sure about the double ">> >>"?

Yeah ... that was totally dumb and a classic copy & paste mistake from my side. :-(

Also: the 'pr triage' action lacks some permissions. I'm going to look into it.